runtime/nls/cfre/cfrerr.nls

```diff
@@ -1362,3 +1362,11 @@ J9NLS_CFR_ERR_CP_ENTRY_INVALID_BEFORE_V55.explanation=Please consult the Java Vi
 J9NLS_CFR_ERR_CP_ENTRY_INVALID_BEFORE_V55.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_CP_ENTRY_INVALID_BEFORE_V55.user_response=Contact the provider of the classfile for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_BAD_BOOTSTRAP_ARGUMENT_ENTRY=The bootstrap argument array contains illegal indices to the constant pool
```

Comment:
Can the message be updated to something like:
`BootstrapMethod (%d) arguments contain invalid constantpool entry at index (%d) of type (%d)`

So that it contains the BSM index, the cp entry and the cp entries type?  It makes it much easier for users to understand what went wrong.

Reply:
I originally introduced 2 parameters (bsmMethodIndex and bsmArgumentIndex) for this message but didn't move forward because the modification of the existing method buildError() incurs a bunch of unnecessary updating at all places.

From the user perspective, I am customizing a new method buildBootstrapMethodError() (still in coding) which only addresses this issue. Will see whether it works good in this case.

Reply:
Got a crash with my new changes. Still investigating to see what happens to my code.